charts/devlake/templates/deployments.yaml

```diff
@@ -72,13 +71,15 @@ spec:
           envFrom:
             - configMapRef:
                 name: {{ include "devlake.fullname" . }}-config
+            {{- if .Values.grafana.oauthEnabled }}
+            - configMapRef:
+                name: {{ include "devlake.fullname" . }}-grafana-oauth-config
+            {{- end }}
             {{- if .Values.option.useConnectionDetailsSecret }}
             - secretRef:
                 name: {{ .Values.option.connectionSecretName }}
             {{- end }}
           env:
-            - name: GF_SERVER_ROOT_URL
```

Comment:
This variable is removed, but I do not see it replaced anywhere. Is it not used anymore?

Reply:
Thanks for pointing it out; I must have made a mistake here. I'll add it back.

The logic is:

- if OAuth isn't enabled (by default), we should get Grafana server root URL from this ENV var;
- else (OAuth enabled), the `GF_SERVER_ROOT_URL` is set in the values `grafana.oauthConfig`, so when OAuth is enabled, we do not want this ENV var to override `grafana.oauthConfig.GF_SERVER_ROOT_URL`.

I will add some if/else logic here.

Thanks again; this would break non-OAuth scenarios if not spotted.